src/manifold/src/quickhull.h

```diff
@@ -658,27 +246,28 @@ class QuickHull {
 
   double m_epsilon, epsilonSquared, scale;
   bool planar;
-  std::vector<vec3> planarPointCloudTemp;
-  VertexDataSource originalVertexData;
+  Vec<vec3> planarPointCloudTemp;
+  VecView<vec3> originalVertexData;
   MeshBuilder mesh;
   std::array<size_t, 6> extremeValues;
   DiagnosticsData diagnostics;
 
   // Temporary variables used during iteration process
-  std::vector<size_t> newFaceIndices;
-  std::vector<size_t> newHalfEdgeIndices;
-  std::vector<std::unique_ptr<std::vector<size_t>>> disabledFacePointVectors;
-  std::vector<size_t> visibleFaces;
-  std::vector<size_t> horizonEdgesData;
+  Vec<size_t> newFaceIndices;
+  Vec<size_t> newHalfedgeIndices;
+  std::vector<std::unique_ptr<Vec<size_t>>> disabledFacePointVectors;
```

Comment:
Maybe reorder this to make `std::vector` at the start/end? Just make it a bit more consistent...